lib/api/CAnomalyJob.cc

```diff
@@ -688,6 +688,18 @@ void CAnomalyJob::outputResults(core_t::TTime bucketStartTime) {
     this->writeOutAnnotations(annotations);
     this->writeOutResults(false, results, bucketStartTime, processingTime);
 
+    if (m_ModelConfig.modelPruneWindow() > 0) {
+        // Ensure that bucketPruneWindow is always rounded _up_
+        // to the next whole number of buckets (this doesn't really matter if we enforce
+        // that the model prune window always be an exact multiple of bucket span in the
+        // corresponding Java code)
+        core_t::TTime bucketPruneWindow{
+            m_ModelConfig.modelPruneWindow() / m_ModelConfig.bucketLength() +
+            (m_ModelConfig.modelPruneWindow() % m_ModelConfig.bucketLength() != 0)};
```

Comment:
There's an implicit `bool` to `long` cast here.  I suspect that will be causing a compiler warning on some platforms - you could check the console log of the Windows build of the PR CI.

If it is then an alternative pattern we use elsewhere would be:

```suggestion
        core_t::TTime bucketPruneWindow{
            (m_ModelConfig.modelPruneWindow() + m_ModelConfig.bucketLength() - 1) / m_ModelConfig.bucketLength()};
```